wall_e/src/resources/cogs/ban.py

```diff
@@ -41,7 +41,7 @@ async def load(self):
         # read in ban_list of banned users
         logger.info('[Ban load()] loading ban list from the database')
         self.ban_list = await BanRecords.get_all_active_ban_user_ids()
-        logger.info(f"[Ban load()] loaded the following banned users: {self.ban_list}")
+        logger.info(f"[Ban load()] loaded {len(self.ban_list)} banned users from database")
```

Comment:
this is a better idea: https://docs.djangoproject.com/en/2.2/ref/models/querysets/#count

Reply:
done